scripts/ci/linux-x86_64-build-test-ci.ini

```diff
@@ -0,0 +1,22 @@
+[constants]
+march = 'core2'
+mtune = 'skylake'
+
+cflags = ['-march='+march, '-mtune='+mtune]
+ldflags = []
+
+[built-in options]
+c_args = cflags
+c_link_args = ldflags
+cpp_args = cflags
+cpp_link_args = ldflags
+wrap_mode = 'nofallback'
+force_fallback_for = 'cglm'
+deprecation_warnings = 'no-error'
```

Comment:
`deprecation_warnings` is a taisei project option, not a builtin. It should go into `[project options]`, otherwise I don't think it'll work. Also, it seems to be missing from the macos config.